packages/connect/src/types/coinInfo.ts

```diff
@@ -32,6 +32,7 @@ export const CoinSupport = Type.Object({
     T1B1: Type.Union([Type.String(), Type.Literal(false)]),
```

Comment:
extract to:

```ts
const xxx: Record<DeviceEnum, Type.?????> = {
   T1B1: Type.Union([Type.String(), Type.Literal(false)]),
    T2T1: Type.Union([Type.String(), Type.Literal(false)]),
    T2B1: Type.Union([Type.String(), Type.Literal(false)]),
    T3T1: Type.Union([Type.String(), Type.Literal(false)]),
}
```

Reply:
The whole thing could be refactored to
```
export const CoinSupport = Type.Composite([
    Type.Object({
        connect: Type.Boolean(),
    }),
    Type.Record(
        Type.KeyOfEnum(DeviceModelInternal),
        Type.Union([Type.String(), Type.Literal(false)]),
    ),
]);
```
which resolves to 
```
type CoinSupport = {
    connect: boolean;
    T1B1: string | false;
    T2T1: string | false;
    T2B1: string | false;
    T3T1: string | false;
}
```